Formula/cocogitto.rb

```diff
@@ -0,0 +1,26 @@
+class Cocogitto < Formula
+  desc "Conventional Commits toolbox"
+  homepage "https://github.com/cocogitto/cocogitto"
+  url "https://github.com/cocogitto/cocogitto/archive/refs/tags/5.2.0.tar.gz"
+  sha256 "99f9dee05597d7721f6d046dbfefba5cb8d1c4ae22ced415f724affb3a6bd0cc"
+  license "MIT"
+
+  depends_on "rust" => :build
+  uses_from_macos "zlib"
+
+  def install
+    system "cargo", "install", "--locked", *std_cargo_args
```

Comment:
```suggestion
    system "cargo", "install", *std_cargo_args
```
`--locked` is part of [`std_cargo_args`](https://rubydoc.brew.sh/Formula.html#std_cargo_args-instance_method).